doc.go

```diff
@@ -84,4 +84,38 @@
 //			fmt.Println("Domain: %s", aErr.Domain())
 //		}
 //	}
+//
+// # Polling Operations
+//
+// If an API call returns an Operation, that means it could take some time to
+// complete the work initiated by the API call. Applications that are interested
+// in the end result of the operation they initiated should wait until the
+// Operation.Done field indicates it is finished. To do this, use the service's
+// Operation client, and a loop, like so:
+//
```

Comment:
Do we want to show the import for `NewOperationsService`?

Reply:
It isn't centrally located, [every API has its own](https://pkg.go.dev/google.golang.org/api@v0.142.0/serviceusage/v1#NewOperationsService). I should include a package alias to indicate that.

Reply:
Done.